tests/helixpublishwitharcade.proj

```diff
@@ -128,11 +194,26 @@
     <XUnitRunnerDll>$CORE_ROOT/xunit.console.dll</XUnitRunnerDll>
   </PropertyGroup>
 
-  <ItemGroup>
-    <HelixCorrelationPayload Include="$(CoreRootDir)" />
-    <HelixWorkItem Include="@(XUnitWrapperDll->'%(FileName)'->Replace('.XUnitWrapper', ''))">
-      <PayloadDirectory>%(FileDirectory)</PayloadDirectory>
-      <Command>$(CoreRun) $(XUnitRunnerDll) %(File) $(XUnitRunnerArgs)</Command>
+  <ItemGroup Condition=" '$(UsesHelixSdk)' == 'true' ">
+    <HelixCorrelationPayload Include="$(CoreRootDirectory)" />
+
+    <Payloads Include="$([System.IO.Directory]::GetDirectories($(PayloadsRootDirectory)))" Condition="Exists('$(PayloadsRootDirectory)')" />
+    <Payloads Update="@(Payloads)">
+      <PayloadGroup>$([MSBuild]::MakeRelative($(PayloadsRootDirectory), %(FullPath)))</PayloadGroup>
```

Comment:
Could we get rid of `PayloadGroup` metadata on `Payloads` and just use `TestGroup`? They seem redundant.

Reply:
No, we can't. PayloadGroup can correspond to TestGroup but it's not always a case. For example, if payload contains more than one assembly than PayloadGroup is what have been specified in TestGrouping file for this assembly.